superset/charts/api.py

```diff
@@ -997,3 +1001,56 @@ def import_(self) -> Response:
         except Exception as exc:  # pylint: disable=broad-except
             logger.exception("Import chart failed")
             return self.response_500(message=str(exc))
+
+    @expose("/time_range/", methods=["GET"])
```

Comment:
Thanks @zhaoyongjie this PR is great! Just curious when do we need call this new API `/time_range/`? i assume most of cases we only call the internal function?

Reply:
The current Superset front-end generates human readable time_range strings or time function expressions (a new feature introduced). We need to convert this string to Start Datetime and End Datetime. These calculations are done on the backend, so we need to introduce a new endpoint.

Reply:
My question is do we need `@expose` decorator: https://flask-admin.readthedocs.io/en/latest/api/mod_base/#base-view. The decorator is used to create a `view`, but i feel what we need here is only an internal function `def time_range`

Reply:
We need to return an explicit time range to the browser, so we need the backend to take this human-readable string convert to start datetime and end datetime.  the explicit time range show on control lable
```
"yesterday : tomorrow"  ===  2020-12-17 <= col < 2020-12-19
```

<img width="214" alt="image" src="https://user-images.githubusercontent.com/2016594/102542838-5c021800-40ed-11eb-8194-4ae28bf9284f.png">